@here/harp-examples/src/textcanvas.ts

```diff
@@ -335,7 +335,7 @@ export namespace TextCanvasDynamicExample {
 
     function main() {
         // Init Three.JS
-        webglRenderer = new THREE.WebGLRenderer({
+        webglRenderer = new THREE.WebGL1Renderer({
```

Comment:
with this change we require all our users to update to 0.118. I'm okay with this but if we want to support users on older versions of three (117?) we could replace this test with something like
```
webglRenderer = new ((THREE as any).WebGL1Renderer ?? THREE.WebGLRenderer)(…)
```